src/components/VStep.vue

```diff
@@ -15,10 +15,10 @@
 
     <slot name="actions">
       <div class="v-step__buttons">
-        <button @click.prevent="stop" v-if="!isLast" class="v-step__button">{{ labels.buttonSkip }}</button>
-        <button @click.prevent="previousStep" v-if="!isFirst" class="v-step__button">{{ labels.buttonPrevious }}</button>
-        <button @click.prevent="nextStep" v-if="!isLast" class="v-step__button">{{ labels.buttonNext }}</button>
-        <button @click.prevent="stop" v-if="isLast" class="v-step__button">{{ labels.buttonStop }}</button>
+        <button @click.prevent="stop" v-if="!isLast" class="v-step__button skip">{{ labels.buttonSkip }}</button>
+        <button @click.prevent="previousStep" v-if="!isFirst" class="v-step__button prev">{{ labels.buttonPrevious }}</button>
+        <button @click.prevent="nextStep" v-if="!isLast" class="v-step__button next">{{ labels.buttonNext }}</button>
+        <button @click.prevent="stop" v-if="isLast" class="v-step__button stop">{{ labels.buttonStop }}</button>
```

Comment:
I think we should name space these classes as they are fairly generic. Any project that already has the class `.next`  with apply those styles to this

Reply:
I'm also not fond of abbreviations in code (`prev` instead of `previous`).
We could use something like `v-step__button-skip` or maybe a modifier `v-step__button--skip` but it's not really one... So I'll rather go with the first option and label those as:

```
v-step__button-skip
v-step__button-previous
v-step__button-next
v-step__button-stop
```

In addition to the initial `v-step__button` class of course to stay BEM compliant.